From 86183e5b6beaa37edbf24fa16a51c5d52edbc8e2 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 8 Jan 2020 13:14:18 +0100
Subject: [PATCH 1/2] Revert "libubox: update to version 2019-12-28"

It is included in OpenWrt 19.07, but it was not properly tested.
There are some bugs, which needs to be reported to OpenWrt developers.
Some services does not start. For example, mostly affected by this
change is fosquitto and because of that Foris does not start.

This reverts commit 04fd5e22b24259725a7932b3be0916c3fa750f64.
---
 package/libs/libubox/Makefile | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/package/libs/libubox/Makefile b/package/libs/libubox/Makefile
index 24b5b45068..8a0533ce4c 100644
--- a/package/libs/libubox/Makefile
+++ b/package/libs/libubox/Makefile
@@ -5,9 +5,9 @@ PKG_RELEASE=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL=$(PROJECT_GIT)/project/libubox.git
-PKG_MIRROR_HASH:=90511058f49a178a816d520944cddba421f9eb50c319ef1623bed7aa22f94a3a
-PKG_SOURCE_DATE:=2019-12-28
-PKG_SOURCE_VERSION:=cd75136b1342e1e9dabf921be13240c6653640ed
+PKG_MIRROR_HASH:=58f6f6b21a00b69d466bcbce31e8adbef364d139715731430194746be5211840
+PKG_SOURCE_DATE:=2019-10-29
+PKG_SOURCE_VERSION:=301303911dded723b7eda4d6a4a933b22d2c1b60
 CMAKE_INSTALL:=1
 
 PKG_LICENSE:=ISC
@@ -27,7 +27,7 @@ define Package/libubox
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=Basic utility library
-  ABI_VERSION:=20191228
+  ABI_VERSION:=20170601
   DEPENDS:=
 endef
 
-- 
2.24.1

